SetupDataPkg/Tools/VariableList.py

```diff
@@ -1278,6 +1277,68 @@ def write_csv(schema, csv_path, full, subknobs=True):
                             knob.format.object_to_string(knob.value),
                             string_binary,
                             knob.help])
+                            
+                            
+def write_csv_detailed(schema, csv_path):
+
+    with open(csv_path, 'w', newline='') as csv_file:
+        writer = csv.writer(csv_file)
+        guid = None
+        namespace_changed = True # Used to print the guid on the first row and then only print

```

Comment:
Is this "*" for guid feature really what you need? This seems helpful for human reading of the output, but if you are going to process this with a script, it would probably be a lot easier if each row was standalone. This may become a bigger issue if/when there are manifests with multiple guids.

Reply:
It's just that I didn't want to make a lot of changes from what you initially had in the write_csv(). It would be easier to have the guid written out to each line individually. I will make the change. Thanks.

Reply:
I think it was added that for easier human readability. For scripting purposes, it just complicates things for the consumer, so if you want to keep it, I'd put it behind an argument.

Reply:
![image](https://github.com/microsoft/mu_feature_config/assets/171983597/67b24d3c-e31e-47c1-9fd0-3aa962a72fc3)

Reply:
The short-term plan is to have this CSV file as the starting point for PnP team to apply any combination of knob settings (rather than start from the ConfigEditor). We might end up developing other scripts that potentially modify this CSV file with some user input for tuning purposes, but that's more of a long-term plan in which case we can switch back to using the compact CSV format. Can you please clarify what you mean by "put it behind an argument"?